airflow/models/dagrun.py

```diff
@@ -724,6 +755,13 @@ def recalculate(self) -> _UnfinishedStates:
                     msg="task_failure",
                 )
 
+            # checking if the max_failure_runs has been provided and last consecutivate failures are more
+            # than this number if so we have to mark this dag as off
```

Comment:
```suggestion
            # than this number if so we have to pause this DAG
```

Lets use the same wording for everything for readability, so instead of "marking off" -> "pausing".